src/components/Card/CardContent.tsx

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
 
+import { SpaceSizes } from '../../theme';
+import { CardContainer } from './Card';
+
 const CardContent = React.forwardRef<
   HTMLDivElement,
   React.PropsWithChildren<Record<string, unknown>>
 >(({ children, ...props }, ref) => (
   <div ref={ref} {...props}>
```

Comment:
Let's move `ref` and `...props` to the `CardContainer` there is no need to add another node into the DOM